### ISSUE TITLE
Slackbot response message handling

### ISSUE THREAD

**Issue by michael-robbins** [emojirades/emojirades#332]
Currently when slackbot itself posts a message it has a different message type:
```
ERROR:EmojiradesBot.slack.SlackClient:Failed to run a message listener: Unable to find Workspace ID in message event: {'type': 'message', 'subtype': 'slackbot_response', 'user': 'USLACKBOT', 'text': ':waddle::waddle2::waddle::waddle2::waddle::waddle2::waddle::waddle2::waddle:', 'channel': 'CABC123', 'event_ts': '1630883262.394800', 'ts': '1630883262.394800'}
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/slack_sdk/rtm_v2/__init__.py", line 336, in run_message_listeners
    listener(self, message)
  File "/usr/local/lib/python3.8/site-packages/slack_sdk/rtm_v2/__init__.py", line 172, in new_message_listener
    func(_self, event)
  File "/usr/local/lib/python3.8/site-packages/emojirades/bot.py", line 70, in handle_event
    raise RuntimeError(
RuntimeError: Unable to find Workspace ID in message event: {'type': 'message', 'subtype': 'slackbot_response', 'user': 'USLACKBOT', 'text': ':waddle::waddle2::waddle::waddle2::waddle::waddle2::waddle::waddle2::waddle:', 'channel': 'CABC123', 'event_ts': '1630883262.394800', 'ts': '1630883262.394800'}

**Comment by michael-robbins** [comment]
I wonder if the RTM Client provides this as an attribute of itself and we can just grab it there vs having to handle all these different message types.

Especially for this one, given the Team ID isn't actually present at all :/

**Comment by michael-robbins** [comment]
Believe I've fixed this in #333
